docs/guides/python-collector.md

```diff
@@ -0,0 +1,481 @@
+        'lines': [
+            ['current_humidity']
+        ]
+     }
+
+}
+```
+
+As before, initiate new dimensions and add data to them:
+
+```python
+    data['current_temperature'] = self.weather_data["temp"]
+    data['min_temperature'] = self.weather_data["min_temp"]
+    data['max_temperature'] = self.weather_data["max_temp"]
+    data['avg_temperature`'] = self.weather_data["av_temp"]
+    data['current_humidity'] = self.weather_data["humid"]
+```
+
+Finally, set the order for the `temp_stats` chart:
+
+```python
+ORDER = [
+    'temp_current',
+    ‘temp_stats’
+    'humid_current'
+]
+```
+
+[Restart Netdata](/docs/configure/start-stop-restart.md) with `sudo systemctl restart netdata` to see the new
+min/max/average temperature chart with multiple dimensions:
+
+![A snapshot of the modified chart](https://i.imgur.com/g7E8lnG.png)
+
+## Add a configuration file
+
+The last piece of the puzzle to create a fully robust Python collector is the configuration file. Python.d uses
+configuration in [YAML](https://www.tutorialspoint.com/yaml/yaml_basics.htm) format and is used as follows:
+
+- Create a configuration file in the same directory as the `<plugin_name>.chart.py`. Name it `<plugin_name>.conf`.
+- Define a `job`, which is an instance of the collector. It is useful when you want to collect data from different
+  sources with different attributes. For example, we could gather data from 2 different weather stations, which use
+  different temperature measures: Fahrenheit and Celcius.
+- You can define many different jobs with the same name, but with different attributes. Netdata will try each job
+  serially and will stop at the first job that returns data. If multiple jobs have the same name, only one of them can
+  run. This enables you to define different "ways" to fetch data from a particular data source so that the collector has
+  more chances to work out-of-the-box. For example, if the data source supports both `HTTP` and `linux socket`, you can
+  define 2 jobs named `local`, with each using a different method. 
+- Check the `postgresql` collector configuration file on
+  [GitHub](https://github.com/netdata/netdata/blob/master/collectors/python.d.plugin/postgres/postgres.conf) to get a
+  sense of the structure.
+
+```yaml
+weather_station_1:
+    name: 'Greece'
+    endpoint: 'https://endpoint_1.com'
+    port: 67
+    type: 'celcius'
+weather_station_2:
+    name: 'Florida: USA'
+    endpoint: 'https://endpoint_2.com'
+    port: 67 
+    type: 'fahrenheit'
+```
+
+Next, access the above configuration variables in the `__init__` function:
+
+```python
+def __init__(self, configuration=None, name=None):
+        SimpleService.__init__(self, configuration=configuration, name=name)
+        self.endpoint = self.configuration.get('endpoint', <default_endpoint>)
+```
+
+Because you initiate the `framework class` (e.g `SimpleService.__init__`)q, the configuration will be
```

Comment:
```suggestion
Because you initiate the `framework class` (e.g `SimpleService.__init__`), the configuration will be
```

Reply:
Thanks for catching these q's I added when I was trying to hit the keyboard shortcut to wrap lines! :laughing: